core/src/main/java/io/neow3j/protocol/core/methods/response/InvocationResult.java

```diff
@@ -49,6 +49,10 @@ public String getState() {
         return state;
     }
 
+    public boolean hasStateFault() {
+        return state.equals("FAULT");
```

Comment:
Is there any possibility that the `state` is `null`? Does it make sense to add this check here?

Reply:
I don't think so. The state is always either `NONE`, `HALT`, `FAULT` or `BREAK`.